capacity-mode-evaluator/src/cost_estimates.py

```diff
@@ -0,0 +1,189 @@
+import pandas as pd
+import numpy as np
+
+
+def cost_estimate(results_metrics_df, results_estimates_df, read_util, write_util, read_min, write_min):
+    metric_map = {
+        'ConsumedWriteCapacityUnits': ('ProvisionedWriteCapacityUnits', write_min, write_util, 0.00065, 1.25/1000000),
```

Comment:
Costs aren't changed from region to region? We [hard code the costs](https://github.com/awslabs/amazon-dynamodb-tools/blob/main/reco/src/ddb_rc_reco/config.py#L9) in other parts of this package. If you're going to quote savings you need to be right. Perhaps we can move the cost matrix from reco up to a top level folder so that your code can use it? Or you can make a package that both tools can use to pull in cost data?

Reply:
added pricing function to retrieve pricing info based on user session region